style.css

```diff
@@ -0,0 +1,834 @@
+.wrap {
+  background: url(./img/backround.jpg) no-repeat;
+  background-size: cover;
+  background-position: top;
+  height: 100vh;
+  margin: 0 auto;
+  font-family: "Museo", Arial, sans-serif;
+  max-width: 1920px;
+  min-width: 375px;
+}
+
+.logo {
+  width: 146.18px;
+  height: 54.84px;
+  margin-right: 91px;
+}
+
+.header {
+  margin: 0 auto;
+  max-width: 89%;
+  height: 21.5vh;
+  display: flex;
+  align-items: center;
+}
+
+.wrap-logo {
+  flex: 1;
+  display: flex;
+  justify-content: center;
+}
+
+.stumb {
+  flex: 1;
+}
+
+.desktop-nav {
+  flex: 1;
+  display: flex;
+}
+
+.desktop-nav__item {
+  text-decoration: none;
+  color: #fff;
+  font-weight: 500;
+  font-size: 22px;
+  cursor: pointer;
+  transition: color 2s ease-out;
+  padding: 0 19px;
+  white-space: nowrap;
+}
+
+.desktop-nav__item:hover {
+  color: gray;
+}
+
+.main-wrap {
+  display: flex;
+}
+
+.info {
+  flex: 1;
+  display: flex;
+  justify-content: space-around;
+  flex-basis: 100%;
+}
+
+.info__content {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  flex-basis: 50%;
+  align-items: center;
+}
+
+.info__img {
+  background: url(img/Bali.png) no-repeat;
+  background-size: 95%;
+  width: 537px;
+  height: 252px;
+  display: flex;
+  justify-content: flex-end;
+  align-items: flex-end;
+  margin-bottom: 49px;
+  margin-right: 99px;
+}
+
+.info__text {
+  font-size: 22px;
+  color: #f8eee4;
+  margin: 25px 0;
+  width: 29vw;
+}
+
+.info__btn {
+  width: 293.76px;
+  height: 61.88px;
+  background: #d4bea3;
+  border-radius: 48px;
+  font-size: 22px;
+  color: #ffffff;
```

Comment:
Codacy found an issue: [Expected "#ffffff" to be "#fff" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/VeronikasBaliProject/pullRequest?prid=5972339)